models/engine/file_storage.py

```diff
@@ -68,3 +68,15 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieves one object"""
+        if cls is None or id is None:
+            return None
+        key = "{}.{}".format(cls, id)
+        if key in self.__objects.keys():
+            return self.__objects[key]
+
+    def count(self, cls=None):
+        """count the number of objects"""
+        return len(self.all(cls))
```

Comment:
what happens if cls is None??

Reply:
def all(self, cls=None) has a condition that accounts for if cls is None.